### DISCUSSION TITLE
点击后持续显示天赋详情

### DISCUSSION THREAD

**Discussion by moshuying** [cangzhang/ChampR#224] [Ideas]
如果能提供点击后持续显示天赋详情而不是鼠标经过时短暂悬浮天赋详情就好了，目前这个软件在国服不太好用，但我依旧喜欢这个自由选择天赋的功能。如果能做到点击后持续显示我自己点天赋就行了。

**Comment by ghs01023** [comment]
同意，作者大大能优化下吗
